kmir/src/tests/unit/test_unit.py

```diff
@@ -1,5 +1,148 @@
-from kmir.hello import hello
+from __future__ import annotations
 
+from typing import TYPE_CHECKING
 
-def test_hello() -> None:
-    assert hello('World') == 'Hello, World!'
```

Comment:
I took the liberty to move all parse tests which don't read files into the `test_unit` module and remove the `hello` test.